### PR TITLE
Added favicon.ico closes #2

### LINKED ISSUE #2: Add favicon
place favicon in `static/favicon.ico`

### PR DESCRIPTION
**Pull request by theycallmemac**
Along with this I've made some changes to both ```content/about/_index.md``` and ```data/schedule.yml```.

### CODE REVIEW

File: content/about/_index.md [hunk 1]
```diff
@@ -35,7 +16,7 @@ Redbrick, DIT Compsoc and DU Netsoc. They recognised that there is no event in
 Ireland targets topic in a way that doesn't go above most students' heads.
 
 A lot of the material at conferences across the country is usually geared
-towards the graduates rather than the undergraduates. SISTEM is run by a team of
+towards the graduates rather than the undergraduates. SISTEM is ran by a team of
```
Comment (GoldenBadger) [inline content/about/_index.md]: I believe this should stay as "run".

**Review by GoldenBadger — CHANGES_REQUESTED** [review]

File: data/schedule.yaml [hunk 2]
```diff
@@ -1,36 +1,12 @@
...
-        time: '09:30'
-      - name: SLOT FREE
-        by: NO ONE
-        time: '11:30'
-      - name: SLOT FREE
-        by: NO ONE
-        time: '14:15'
   - location: Conference Room
     description: Main Track
+    live: true
```
Comment (GoldenBadger) [inline data/schedule.yaml]: What does this tag mean?

**Review by GoldenBadger — APPROVED** [review]
LGTM

File: data/schedule.yaml (hunk 2 above)
Comment (theycallmemac) [reply data/schedule.yaml]: Not sure, cannot actually recall adding that. @butlerx any ideas?

Comment (butlerx) [reply data/schedule.yaml]: it was a way to say there would be a live stream. since replaced with checking for video link. so no longer needed

Comment (theycallmemac) [reply data/schedule.yaml]: okay so, just removed it.

**Review by butlerx — APPROVED** [review]

**Pull request merged**